main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        toaster
+//  action 1:    turn on heating element
+toaster.activateHeat();
+//  action 2:    turn off heating element
+toaster.deactivateHeat();
+//  action 3:    eject toast
+toaster.eject();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        latch
+//  action 1:    set to on position
+latch.setOn();
+//  action 2:    set to off position
+latch.setOff();
+//  action 3:    toggle position
+latch.toggle();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        plant
+//  action 1:    convert sunlight to energy
+plant.photosynthesize();
+//  action 2:    grow fruit
+plant.growFruit();
+//  action 3:    drop fruit
+plant.dropFruit();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        ceiling fan
+//  action 1:    toggle light bulb power
+ceilingFan.toggleLight();
+//  action 2:    cycle through fan power states
+ceilingFan.incrementFanPowerState();
+//  action 3:    detach from ceiling
+ceilingFan.detach();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        fire alarm
+//  action 1:    activate siren
+fireAlarm.activateSiren();
+//  action 2:    activate lights
+fireAlarm.activateLights();
+//  action 3:    notify fire department
+fireAlarm.notifyFireDept();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        goat
+//  action 1:    eat
+goat.eat();
+//  action 2:    vocalize
+goat.vocalize();
+//  action 3:    fart
+goat.fart();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        rocket
+//  action 1:    launch
+rocket.launch();
+//  action 2:    detach primary thrusters
+rocket.detachPrimaryThrusters();
+//  action 3:    activate secondary thrusters
+rocket.activateSecondaryThrusters();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        star
+//  action 1:    emit radiation
+star.emitRadiation();
+//  action 2:    spawn solar flare
+star.spawnFlare();
+//  action 3:    supernova
+star.supernova();
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        dog
+//  action 1:    bark
+dog.bark();
+//  action 2:    poop
+dog.poop();
+//  action 3:    zoom
+dog.zoom();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.